tests/test-requirements.txt

```diff
@@ -2,6 +2,6 @@ pytest
 pytest-timeout
 hypothesis
 requests_mock
-llvmlite ~= 0.40.0
-pytket-qir == 0.4.0
+llvmlite ~= 0.41.1
+pytket-qir == 0.6.0
```

Comment:
Doing this update will not work at the moment, the llvmlite update should be fine